test-framework/common/src/main/java/io/quarkus/test/common/RestAssuredURLManager.java

```diff
@@ -92,6 +91,15 @@ public void setURL() {
         }
     }
 
+    public static String getHost() {
```

Comment:
You should not need these changes. Just inject the URL using @TestHTTPResource and then get the host and port from the URL.